src/Views/Admin/AddTeamComponents/MembersCard.jsx

```diff
@@ -0,0 +1,59 @@
+import MemberProf from "../../../assets/MemProf.png";
+import { BsTwitch } from "react-icons/bs";
+import { BsFacebook } from "react-icons/bs";
+import { BsDiscord } from "react-icons/bs";
+
+const anchorStyle = {
+  textDecoration: "none", // Remove underline
+  color: "inherit", // Use the default text color
+};
+
+const MembersCard = ({
+  memberName,
+  memberProfileType,
+  memberAddress,
+  discordLink,
+  fbLink,
+  twitchLink,
```

Comment:
Pass only the variable names which contains all data from the map since they belong to just one entity.